### ISSUE TITLE
Plugin 2022.3 causes a stacktrace in IDEA 2022.2 when opening a project

### ISSUE THREAD

**Issue by jashaj** [meanmail-dev/requirements#91]
**Describe the bug**

The plugin causes a stacktrace when I open a project. This stacktrace is not present when I uninstall the plugin.

**To Reproduce**

- Update to IntelliJ 2022.2
- Open IntelliJ
- Open a project
- An error is shown in the notifications tab

**Expected behavior**

No error when opening a project 

**IDE (please complete the following information. See Main Menu -> Help -> About):**

 - OS: macOS 12.5
 - Product: Idea Ultimate
 - Version IU-222.3345.118 (2022.2)
- Runtime version: 17.0.3+7-b469.32 x86_64  
  VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
 - Project SDK: 
   - No SDK
   - Java 11
   - Python 3.9

**Plugin (please complete the following information. See Main Menu -> File -> Settings -> Plugins):**
 - Version 2022.3-222-EAP-SNAPSHOT

**Additional context**

```
Detector ru.meanmail.fileTypes.RequirementsFileTypeDetector@5a07b546 (class ru.meanmail.fileTypes.RequirementsFileTypeDetector) exception occurred:

java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.vfs.VirtualFile.getName()" because the return value of "com.intellij.openapi.vfs.VirtualFile.getParent()" is null
	at ru.meanmail.fileTypes.RequirementsFileTypeDetector.detect(RequirementsFileTypeDetector.kt:12)
	at com.intellij.openapi.fileTypes.impl.FileTypeDetectionService.lambda$detectFromContent$5(FileTypeDetectionService.java:557)
	at com.intellij.openapi.fileEditor.impl.LoadTextUtil.processTextFromBinaryPresentationOrNull(LoadTextUtil.java:640)
	at com.intellij.openapi.fileTypes.impl.FileTypeDetectionService.detectFromContent(FileTypeDetectionService.java:546)
	at com.intellij.openapi.fileTypes.impl.FileTypeDetectionService.detectFromContentAndCache(FileTypeDetectionService.java:516)
	at com.intellij.openapi.fileTypes.impl.FileTypeDetectionService.getOrDetectFromContent(FileTypeDetectionService.java:250)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.internalContinueToDetectFileTypeByFile(FileTypeManagerImpl.java:759)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.getFileTypeByFile(FileTypeManagerImpl.java:750)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.getFileTypeByFile(FileTypeManagerImpl.java:687)
	at com.intellij.openapi.vfs.VirtualFile.getFileType(VirtualFile.java:356)
	at com.intellij.psi.SingleRootFileViewProvider.fileSizeIsGreaterThan(SingleRootFileViewProvider.java:200)
	at com.intellij.psi.SingleRootFileViewProvider.isTooLargeForIntelligence(SingleRootFileViewProvider.java:169)
	at com.intellij.psi.SingleRootFileViewProvider.isTooLargeForIntelligence(SingleRootFileViewProvider.java:156)
	at com.goide.psi.GoIntelligenceLimitPusher.<init>(GoIntelligenceLimitPusher.java:27)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:844)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.instantiateClass(XmlExtensionAdapter.kt:79)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter$SimpleConstructorInjectionAdapter.instantiateClass(XmlExtensionAdapter.kt:90)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:50)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.kt:31)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapter(ExtensionPointImpl.java:475)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:423)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.computeExtensionList(ExtensionPointImpl.java:251)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.java:245)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensionList(ExtensionPointName.java:39)
	at com.intellij.util.indexing.UnindexedFilesUpdater.collectIndexableFilesConcurrently(UnindexedFilesUpdater.java:459)
	at com.intellij.util.indexing.UnindexedFilesUpdater.scan(UnindexedFilesUpdater.java:235)
	at com.intellij.util.indexing.UnindexedFilesUpdater.scanAndUpdateUnindexedFiles(UnindexedFilesUpdater.java:278)
	at com.intellij.util.indexing.UnindexedFilesUpdater.lambda$performScanningAndIndexing$19(UnindexedFilesUpdater.java:628)
	at com.intellij.util.gist.GistManagerImpl.runWithMergingDependentCacheInvalidations(GistManagerImpl.java:120)
	at com.intellij.util.indexing.UnindexedFilesUpdater.performScanningAndIndexing(UnindexedFilesUpdater.java:627)
	at com.intellij.util.indexing.UnindexedFilesUpdater.performInDumbMode(UnindexedFilesUpdater.java:617)
	at com.intellij.openapi.project.DumbServiceMergingTaskQueue$QueuedDumbModeTask.executeTask(DumbServiceMergingTaskQueue.java:211)
	at com.intellij.openapi.project.DumbServiceMergingTaskQueue$QueuedDumbModeTask.executeTask(DumbServiceMergingTaskQueue.java:195)
	at com.intellij.openapi.project.DumbServiceGuiTaskQueue.lambda$runSingleTask$1(DumbServiceGuiTaskQueue.java:64)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
	at com.intellij.openapi.project.DumbServiceGuiTaskQueue.runSingleTask(DumbServiceGuiTaskQueue.java:62)
	at com.intellij.openapi.project.DumbServiceGuiTaskQueue.lambda$processTasksWithProgress$0(DumbServiceGuiTaskQueue.java:49)
	at com.intellij.util.io.storage.HeavyProcessLatch.performOperation(HeavyProcessLatch.java:76)
	at com.intellij.openapi.project.DumbServiceGuiTaskQueue.processTasksWithProgress(DumbServiceGuiTaskQueue.java:49)
	at com.intellij.openapi.project.DumbServiceImpl.lambda$runBackgroundProcess$13(DumbServiceImpl.java:614)
	at com.intellij.openapi.util.ShutDownTracker.executeWithStopperThread(ShutDownTracker.java:114)
	at com.intellij.openapi.project.DumbServiceImpl.runBackgroundProcess(DumbServiceImpl.java:610)
	at com.intellij.openapi.project.DumbServiceImpl$5.run(DumbServiceImpl.java:587)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:442)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsynchronously$5(CoreProgressManager.java:493)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:252)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:252)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by vvvhell** [comment]
same here... Any news on when we can expect a fix?

**Comment by meanmail** [comment]
Fixed in 2022.3.2
